benchmarks/continous/db/tool/orm/src/models.rs

```diff
@@ -0,0 +1,23 @@
+use chrono::{DateTime, Utc};
+use diesel::prelude::Insertable;
+
+use crate::schema::ft_transfers;
+
+#[derive(Insertable)]
+#[diesel(table_name = ft_transfers)]
+pub struct NewFtTransfer<'a> {
```

Comment:
This is not going to be a performance critical code - so I would err on the side of making it simpler and more maintainable by copying the necessary data and removing the need for ownership annotations.

Reply:
Addressed in 1ea8d91.